drivers/hud_apr/spec/models/fy2023/datalab_apr/rrh_coc_1.rb

```diff
@@ -28,9 +28,9 @@
         file_path: result_file_prefix + 'apr/rrh_coc_1',
         question: 'Q5a',
         # Pending AAQ: [APR] DataLab - RRH CoC I & DataLab - RRH CoC II (D) Q5 B16
-        skip: [
-          'B16',
-        ],
+        # skip: [
+        #   'B16',
+        # ],
```

Comment:
If B16 passes, should this be removed?

I guess this question applies in general, anywhere where you commented out the skip, it seems like we should just clean it up...

I guess the AAQs for the old test kit are no longer relevant, whether they were answered or not :-)